examples/OnDemandResources Example/TestLibrary/TestLibrary.podspec

```diff
@@ -47,15 +47,15 @@ TODO: Add long description of the pod here.
     app_spec.source_files = 'App1/Classes/**/*'
 
     app_spec.on_demand_resources = {
-      'a1' => ['App1/app1_on_demand_bundle1/*']
+      'a1' => { :paths => ['App1/app1_on_demand_bundle1/*'], :category => :initial_install }
```

Comment:
these do not show up above in project diff because they are `app_spec`s which only live in Pods.xcodeproj which for examples are not tracked on git.